http/http_test.go

```diff
@@ -0,0 +1,208 @@
+package http_test
+
+import (
+	"bytes"
+	"context"
+	"html/template"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+
+	dskitHttp "github.com/grafana/dskit/http"
```

Comment:
The norm with import aliases is to write them in all lowercase I believe.
```suggestion
	dskithttp "github.com/grafana/dskit/http"
```

Reply:
Ah goland offers a refactor to camel case, so I usually take that short route.